packages/idx/src/property/list.component.html

```diff
@@ -39,16 +39,16 @@
                     </a>
                 </div>
                 <div class="property-content">
-                    <md-card-title class="property-address dotted-spaced-underline">
-                        <md-card-title-text aria-label="Address" id="{{::elementId}}_{{::property._id}}_address">
+                    <div class="property-address dotted-spaced-underline">
```

Comment:
It all becomes divs...

Reply:
would be nice at some point to change every divider element we have into a properly named element for organization